spec/features/login_redirects_to_posts_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.feature "Redirect to Posts", type: :feature do
+  scenario "Can see posts after logging in" do
```

Comment:
This should really say that you are on the posts page, not that you can see the posts